examples/nirfmxnr/acp-single-carrier.py

```diff
@@ -69,7 +69,7 @@
 def raise_if_error(response):
     if response.status != 0:
         error_response = client.GetError(
-            nirfmxwlan_types.GetErrorRequest(
+            nirfmxnr_types.GetErrorRequest(
```

Comment:
It's interesting/frustrating that `mypy` didn't catch these.

Reply:
IIRC, mypy only checks functions that are annotated, so we'd need to put in typing info for the `response` parameter and/or the return value.